app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -26,7 +27,9 @@ class ProfileEditActivityPresenter @Inject constructor(
   @Inject
   lateinit var profileEditViewModel: ProfileEditViewModel
 
-  fun handleOnCreate() {
+  private var dialog: AlertDialog? = null
```

Comment:
You can use `lateinit` like this:
`private lateinit var dialog: AlertDialog`

Also, this will bring one more change
```
  fun handleOnSaveInstanceState(outState: Bundle) {
    if (::dialog.isInitialized) {
      outState.putBoolean(IS_PROFILE_DELETION_DIALOG_VISIBLE_KEY, dialog.isShowing)
    } else {
      outState.putBoolean(IS_PROFILE_DELETION_DIALOG_VISIBLE_KEY, false)
    }
  }
```